types/d3-polygon/d3-polygon-tests.ts

```diff
@@ -27,7 +28,9 @@ num = d3Polygon.polygonArea(polygon);
 
 point = d3Polygon.polygonCentroid(polygon);
 
-hull = d3Polygon.polygonHull(pointArray);
+hull = d3Polygon.polygonHull(pointArray)!;
```

Comment:
Would it make more sense to turn this into an `$ExpectError` test since the type of `hull` does not cover all the union cases of the `polygonHull()` function? I'm not sure what the guidelines for using the non-null assertion operator are, but it seems like it would be a better idea to show the failure than hide the type mismatch.

Reply:
I wrote this test as a typical code example. But this is indeed not a great test for the definition. It mostly tests the "non-null assertion operator" behaviour and not d3 behaviour.

About using `$ExpectError` in this particular case, I think it make little sens to test if it returns `Array | null` and then tests also if it does _not_ return `Array`. As one imply the other. So I removed the test.